apps/logo-quiz/src/app/views/LevelList/components/LevelPreview/LevelPreview.scss

```diff
@@ -1,7 +1,71 @@
 .level-button {
   display: block;
   border: none;
-  background: lightblue;
-  margin-bottom: 10px;
   min-width: 200px;
+  font-size: 1.25em;
+  font-weight: 600;
+  color: #1D3557;
```

Comment:
should we create a file with all colors used as variables?

Reply:
Yeah. I've yet to figure out how to do that in React, especially how to configure css path mappings as we normally did with Angular's `angular.json`. I'll leave this for later, if that's fine with you, just to move on.